inc/class-upgrade.php

```diff
@@ -133,6 +136,17 @@ public function __construct() {
 		$this->finish_up();
 	}
 
+	/**
+	 * Adds a new upgrade history entry.
+	 *
+	 * @param string $current_version The old version from which we are upgrading.
+	 * @param string $new_version     The version we are upgrading to.
+	 */
+	protected function add_upgrade_history( $current_version, $new_version ) {
+		$upgrade_history = new WPSEO_Upgrade_History();
+		$upgrade_history->add( $current_version, $new_version, array_keys( WPSEO_Options::$options ) );
```

Comment:
We might wanna exclude 'wpseo_ms' here somehow, since the class should only be responsible for sitewide options at this point. On the other hand, it only looks in the `wp_options` table, so that option being in the array likely doesn't have any effect either way. Just something to think about.

Reply:
We'll just keep it in there.
For the developer picking this up, please run a test on a multisite environment to see what is stored when upgrading on Network Admin and subsite.